charts/sourcegraph/templates/codeintel-db/codeintel-db.Service.yaml

```diff
@@ -18,9 +18,9 @@ metadata:
   name: codeintel-db
 spec:
   ports:
-  - name: pgsql
+  - name: codeintel-db
     port: 5432
-    targetPort: pgsql
+    targetPort: codeintel-db
```

Comment:
the target port does not seem rigth to me: 

https://github.com/sourcegraph/deploy-sourcegraph-helm/blob/2b4383b5c617d534a1a35fa79910cedb84850e2a/charts/sourcegraph/templates/codeintel-db/codeintel-db.StatefulSet.yaml#L92-L94

shouldn't it be `psql`?

Reply:
Shouldn't `targetPort` always match the `name` -- just for readability? Why would this case be different than the `codeinsights-db` case of the service manifest?

Reply:
Any design reason we named the db container in codeintel-db `pgsql` that we know of?

Reply:
in svc port entry

```yaml
spec:
  selector:
    app.kubernetes.io/name: MyApp
  ports:
    - protocol: TCP
      port: 80
      targetPort: 9376
```

it basically says, forward any incoming traffic to this svc to pod with matching selector labels at `9376`

so targetPort should match the port name or number at the __POD__

Reply:
if we really want to use `codeintel-db` everywhere, we will need to rename the port name in statefulset pod template s well